extensions/adobe/experience/decisioning/decisionevent.schema.json

```diff
@@ -20,7 +20,40 @@
   "description": "A decision event is used to capture observations about the outcome and context of a decision activity. The decision event captures information about how the decision made, when it was occurring, what options were proposed (chosen) and what contextual state existed that either influenced the decision or could be observed during the decision process. The decision event also captures the proposition ID, a globally unique identifier that can be used to correlate the decision to other events. DecisionEvents are not only relatable to Experience Events that impacted a decision but also to ExperienceEvents that are a direct response to a proposition. It is the expectation that applications reference the proposition ID in every experience event that was influenced by the propositions. A user profile's proposition-response history is maintained with these proposition IDs.",
   "definitions": {
     "decisionevent": {
-      "properties": {}
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/propositions": {
+          "title": "Proposition Details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/proposition-details"
+          },
+          "description": "Details about the results of the decision. An array providing the output of a single decision event, one proposition per activity parameter in the request."
+        },
+        "https://ns.adobe.com/experience/decisioning/experienceID": {
+          "title": "Experience Reference",
+          "type": "string",
+          "description": "Unique identifier for the proposition's content. Recipients of propositions with the same value will receive the same content experience in the targeted placements. Moreover, propositions with the same value to the same user mean that the content of the propositions has not changed. Propositions with values indicate that the decision selections are different and a different experience resulted from the decision. Note that adding a placement to the decision scope always changes the experience value."
+        }
+      }
+    },
+    "proposition-details": {
```

Comment:
Why not just include a proposition-interaction here? You have created a differently named structure with most of the proposition-interaction schema.